website/client/components/inventory/equipment.vue

```diff
@@ -65,12 +65,15 @@
           v-for="(label, group) in gearTypesToStrings",
           :key="group",
           :item="flatGear[activeItems[group]]",
+          :showPopover="flatGear[activeItems[group]] && flatGear[activeItems[group]].key.indexOf('_base_0') === -1",
           :label="$t(label)",
           :selected="true",
           :popoverPosition="'top'",
           :starVisible="!costume || user.preferences.costume",
           @click="equip",
         )
+          div(slot="itemContent", :class="'shop_' + flatGear[activeItems[group]].key")
```

Comment:
Like here (referring to https://vuejs.org/v2/guide/components.html#Scoped-Slots)